### DISCUSSION TITLE
i'm having issue with usePrepareContractWrite

### DISCUSSION THREAD

**Discussion by udobuzor** [wevm/wagmi#860] [Q&A]
TypeError: (0 , wagmi__WEBPACK_IMPORTED_MODULE_2__.usePrepareContractWrite) is not a function

const { isConnected} = useAccount();
         const { config } = usePrepareContractWrite ({
         addressOrName: "0xE6d356c99402b78B5345eff7040a9e363Bc5f2C8",
         contractInterface: nftcontractabi,
         functionName: "mint",

**Comment by arlgnv** [comment]
I think you need to open an issue with it and provide more information

**Reply by udobuzor** [reply]
Okay

**Reply by arlgnv** [reply]
If I was helpful you can mark my reply as answer)
